python/cudf/cudf/core/column/column.py

```diff
@@ -752,7 +752,14 @@ def fillna(self, value):
     def isnull(self):
         """Identify missing values in a Column.
         """
-        return libcudf.unary.is_null(self)
+        result = libcudf.unary.is_null(self)
+
+        if self.dtype.kind == "f":
+            # Need to consider `np.nan` values incase
+            # of a float column
+            result = result.binary_operator("or", libcudf.unary.is_nan(self))
```

Comment:
This `.binary_operator` calls will go away as part of https://github.com/rapidsai/cudf/issues/6192